### ISSUE TITLE
{{currentYear}} in `v6.0.0` attributions

### ISSUE THREAD

**Issue by pascalwengerter** [FortAwesome/Font-Awesome#18656]
**Describe the bug**
Using attributions in the `v6.0.0` release produces the console output below, including **{{currentYear}}**, literally :)

```
Font Awesome Free 6.0.0 by @fontawesome - https://fontawesome.com
License - https://fontawesome.com/license/free (Icons: CC BY 4.0, Fonts: SIL OFL 1.1, Code: MIT License)
Copyright {{currentYear}} Fonticons, Inc.
```

**To Reproduce**
1. Add `"@fortawesome/fontawesome-free": "^6.0.0",` to your dependencies
2. Add `import '@fortawesome/fontawesome-free/attribution'` to your app
3. Check logs

**Expected behavior**
Current year gets displayed correctly

**Version and implementation**
Version: `v6.0.0`
Browser and version: Any

**Bug report checklist**
- [X] I have filled out as much of the above information as I can
- [X] I have included a test case because my odds go _way_ up that the team can fix this when I do
- [X] I have [searched for existing issues](https://github.com/FortAwesome/Font-Awesome/issues) and to the best of my knowledge this is not a duplicate

**Comment by robmadole** [comment]
Doh! We'll get that fixed up in the next release.

**Comment by pascalwengerter** [comment]
@robmadole @jasonlundien don't have to time to check it out but I suppose this may have been fixed in the meantime? :)

**Comment by tagliala** [comment]
Yes thanks, fixed in 6.1.0
